submissions/Liddell777/hooli-style-popup/style.css

```diff
@@ -90,18 +90,22 @@ header {
 
 .menu__input {
     position: absolute;
-    width: 30px;
+    position: absolute;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    /* width: 30px;
```

Comment:
For the future, avoid adding commented lines in commit without necessary reason.

Reply:
ok, thanks